runtime/optwin.vim

```diff
@@ -261,6 +261,10 @@ call <SID>OptionG("sect", &sect)
 call append("$", "path\tlist of directory names used for file searching")
 call append("$", "\t(global or local to buffer)")
 call <SID>OptionG("pa", &pa)
+if exists("+cdhome")
```

Comment:
I think this is always true. I've opened <https://github.com/vim/vim/issues/9325>.